core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1130,14 +1130,68 @@ public static String charFromUtf8(int n) {
       assert codePoint instanceof Number;
       long cp = ((Number) codePoint).longValue();
       if (cp < 0 || cp > 255) {
-        throw RESOURCE.inputArgumentsOfCodePointsToBytesOutOfRange(cp).ex();
+        throw RESOURCE.inputArgumentsOfFunctionOutOfRange("CODE_POINTS_TO_BYTES", cp).ex();
```

Comment:
Would you mind checking if there are other functions that have a similar error (input out of range?)? I think maybe some of the recently added regex functions have a similar error. It could be a good idea to reduce some redundancy if that's the case.